public/components/trace_analytics/components/traces/span_detail_table.tsx

```diff
@@ -99,37 +88,22 @@ const renderCommonCellValue = ({
   columnId,
   items,
   tableParams,
-  expandedRows,
-  toggleRowExpansion,
   props,
-  flattenedItems,
-  indentationFactor = 0,
-  fullScreenMode = false,
+  disableInteractions,
 }: {
   rowIndex: number;
   columnId: string;
-  items: any;
+  items: any[];
```

Comment:
Can we remove the `any` from items? we should know the response format here.

Reply:
```
items: Span[];

interface Span {
  spanId: string;
  parentSpanId?: string;
  children: Span[];
  [key: string]: any;
}
```